.vscode/launch.json

```diff
@@ -0,0 +1,37 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
```

Comment:
Added this file, as it contains config to debug tests and samples which will be helpful to other users.

Reply:
I'm not sure if it is a good idea to commit .vscode or some other IDE files